src/runtime_src/core/common/query_requests.h

```diff
@@ -3707,11 +3708,13 @@ struct performance_mode : request
       case 0:
```

Comment:
Please use all lowercase as we are treating them as keywords.

Reply:
The parse_status function is like a to_string_pretty, we print this out directly in reports so we need the first letter capitalized